std_msgs/Header header
#位姿信息
float32 roll
float32 pitch
float32 yaw

#GPS位置
float64 latitude
float64 longitude
float64 altitude

#气压计高度
float32 baro_altitude
#激光测高
float32 tf_altitude

#三轴速度
float32 velocity_n
float32 velocity_e
float32 velocity_d

#目标点信息
float64 g_latitude
float64 g_longitude
float64 g_altitude
